### ISSUE TITLE
DFS Traversal

### ISSUE THREAD

**Issue by ArqumShaikh** [WebClub-NITK/Hacktoberfest-2k19#17]
### Description
Write an efficient code to print the Depth First Search (DFS) traversal of a graph in C++. 

### Details
- Type of issue: Single
- Time Limit: 2 Days
### Issue requirements / progress
- [ ] Read about Depth first search traversal for a graph.
- [ ] Implement DFS using adjacency matrix.
- [ ] Change above code to work with adjacency list.

### Resources
[DFS Tutorial](https://www.hackerearth.com/practice/algorithms/graphs/depth-first-search/tutorial/)

### Directory Structure
Create a new folder called "dfs" inside algorithms folder and add all code related to this issue in the development/dfs folder 

#### Note
Please claim the issue first by commenting here before starting to work on it.

#### For co-ordinators/maintainers creating the issues
For any further clarification on this issue, you are free to contact me,@ArqumShaikh and also you can drop me a request for review! Happy learning!

**Comment by Mksiyak** [comment]
i will work on it.

**Comment by gaurivn** [comment]
I would like to work on this issue.

**Comment by Kuljeet-123** [comment]
I would like to work on it @ArqumShaikh

**Comment by Kuljeet-123** [comment]
@ArqumShaikh  please review my PR and merge into your repo and my PR id is #266.

**Comment by ArqumShaikh** [comment]
@Mksiyak I am assigning this issue to you. Try to get it done before the deadline.

**Comment by SaurabhAgarwala** [comment]
Hacktoberfest 2019 ends. Closing the issue.
